thermo/gpumd/data.py

```diff
@@ -179,6 +182,74 @@ def __modal_analysis_read(nbins, nsamples, datapath,
 # Data-loading Related
 #########################################
 
+def load_compute(directory=None, filename='compute.out', quantities=None):
+    """
+    loads data from compute.out GPUMD output file 
+
+    Args:
+        directory (str):
+            Directory to load 'compute.out' file from (dir. of simulation)
+
+        filename (str):
+            file to load compute from
+
+        quantities (str):
+            allows user to set which quantities to extract from compute.out (such as temperature, force, jp, jk, etc.)
+
+        Returns:
+            'output' dictionary containing the data from compute.out
+    """
+
+    # test 1 was completed in 0.020443599999907747 seconds
+    # test 2 was completed in 2.4768514999999987 seconds
+    if not directory:
+        com_n = os.path.join(os.getcwd(), filename)
+    else:
+        com_n = os.path.join(directory, filename)
+
+    com_n = pd.read_csv(com_n, sep="\s+", header=None)
+
+    total_cols = len(com_n.columns)
+    q_count = {'temperature': 1, 'potential': 1, 'force': 3, 'virial': 3, 'jp': 3, 'jk': 3}
+    output = dict()
+
+    count = 0
+    for value in quantities:
+        count += q_count[value]
+    if 'temperature' in quantities:
+        m = int((total_cols - 2) / count)
+    if 'temperature' not in quantities:
+        m = int(total_cols / count)
```

Comment:
This block does not need two 'if' statements and should use an 'if-else'. Below is an additional way to avoid two 'if' statements:
```suggestion
    m = int(total_cols / count)
    if 'temperature' in quantities:
        m = int((total_cols - 2) / count)
        
```

Reply:
Actually, is there ever a case where `m == 1` is false? If not, then you should remove the variable completely.